website/blog/2022-09-28-analyst-to-ae.md

```diff
@@ -0,0 +1,178 @@
+
+The Analyst caught the issue because they have the appropriate context to validate the dataset. Analysts work directly with the stakeholder, so they’ll also be more familiar with company-specific and department-specific acronyms.
+
+**And as we all know, data validation is an art, not a science.** Analysts will employ everything from a “sniff test” (view a random sample of rows) to individual examples (one-to-one exact matching to another system). An Analyst will have to use their experience to know when the dataset is “good enough” for the stakeholder and their question, since 100% accuracy might not be the goal. And if we're being honest, sometimes being directionally correct is all that’s needed to make a business decision.
+
+An Analyst is able to identify which areas do *not* need to be 100% accurate, which means they can also identify which areas *do* need to be 100% accurate.
+
+> dbt makes it very quick to add [data quality tests](https://docs.getdbt.com/docs/building-a-dbt-project/tests). In fact, it’s so quick, that it’ll take an Analyst longer to write up what tests they want than it would take for an Analyst to completely finish coding them.
+
+When data quality issues are identified by the business, we often see that Analysts are the first ones to be asked:
+
+- Why are over half of the `User_Ids` now showing as NULL?
+- Why is this chart showing a User’s `Location` that’s not in the USA?
+- Why is the dashboard filter saying that all User’s `Zones` are Zone 2?
+
+This familiarity with what types of data quality issues are the most important to the business means that an Analyst can often identify in advance which automated tests should be added to a data model.
+
+## Loop 3: Reacting to mismatched documentation
+
+There’s nothing worse than coming back to view a curated dataset months later (or maybe after someone on the team has offboarded) and learning that there is nothing written to explain why certain logic exists. Or worse, the documentation exists but it no longer matches what the model actually does. Which brings us to our third and final point:
+
+<h3 align="center">Analysts understand the pain of a poorly documented dataset.</h3>
+
+It’s much easier to keep to a naming guide when the writer has a deep understanding of the data and how it’s referenced by the business. Analysts are often asked repeat questions by stakeholders, which is extra incentive to create great documentation from the beginning.
+
+> **A Data Engineer will document a dataset based on what they needed to know to build it. An Analyst with Analytics Engineering skills will document a dataset based on *how to use it going forward.***
```

Comment:
This callout is bolded, but the above one isn't.  I feel like they should match?